test-cases/performance/perf-scylla-cassandra-regular.yaml

```diff
@@ -0,0 +1,51 @@
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=234375001..312500000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=312500001..390625000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=390625001..468750000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=468750001..546875000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=546875001..625000000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=625000001..703125000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=703125001..781250000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=781250001..859375000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=859375001..937500000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=937500001..1015625000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1015625001..1093750000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1093750001..1171875000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1171875001..1250000000"
```

Comment:
Use the regular 1TB dataset we use, it should be faster and less data.
Also, it's using LCS as Dor wanted, but we may want to add ICS option as well in configuration file.

Reply:
done